backend/.env

```diff
@@ -0,0 +1 @@
+BLABLA=eti
```

Comment:
backend/.env file is usually not checked into the repo because it might contain secrets. It's better to check in the template file (.env.template) and explain in the readme what it should contain.